App.js

```diff
@@ -0,0 +1,58 @@
+import Header from "./layouts/header/header";
+import Footer from "./layouts/footer/footer";
+import HomePage from "./pages/homepage/Homepage";
+import Chatbot from './pages/Chatbot/Chatbot';
+import Helloo from "./pages/hello/hello";
+import aboutUs from "./pages/about/aboutUs";
+import Blog from "./pages/blog/blog";
+import Projects from "./pages/projects/projects";
+import News from "./pages/news/news";
+import Contact from "./pages/contact/contact";
+import IotResources from "./pages/resources/iotResources";
+import IotTechnologies from "./pages/resources/iotTechnologies";
+import IotStatistics from "./pages/resources/iotStatistics";
+import IotUpdates from "./pages/resources/iotUpdates";
+import Main from "./pages/faq/main";
+import Report from "./pages/report/report";
+import forgot from "./pages/forgot_password/forgot";
+import Login from "./pages/login/login";
+import Signup from "./pages/signup/signup";
+import './App.css';
+import './Media.css';
```

Comment:
It is not ideal to import custom CSS on the App.js file as this page is React's entry point. It is only ok to do this when you are applying Global styling that is required for the entire web app.